alacritty/src/renderer/text/builtin_font.rs

```diff
@@ -960,6 +883,39 @@ impl Canvas {
     }
 }
 
+/// Size of the stroke to use.
+fn calculate_stroke_size(width: usize) -> usize {
+    // Use one eight of the cell width, since this is used as a step size for block elements.
+    cmp::max((width as f32 / 8.).round() as usize, 1)
+}
+
+/// Equation for `f(x) = slope * x + offset.
+struct LineEquation {
```

Comment:
This is a bad name. It's not an equation it's an iterator. An equation would map to a function in programming so it would just be `fn f(x: i32) -> i32` or whatever.